### PR TITLE
Disable some migration test until vsphere csi driver bug is fixed

### PR DESCRIPTION
**Pull request by gnufied**

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm
/approve
/hold

holding until we see the skip do what it is supposed to do.

File: ci-operator/step-registry/openshift/e2e/vsphere/csi/migration/openshift-e2e-vsphere-csi-migration-workflow.yaml [hunk 1]
```diff
@@ -20,5 +20,13 @@ workflow:
...
         Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (default fs)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (default fs)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (ext4)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (default fs)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Pre-provisioned PV (ext4)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (ext4)] volumes should allow exec of files on the volume
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (ext4)] volumes should store data
+        In-tree Volumes [Driver: vsphere] [Testpattern: Inline-volume (default fs)] volumes should store data
```
Comment (bertinatto) [inline ci-operator/step-registry/openshift/e2e/vsphere/csi/migration/openshift-e2e-vsphere-csi-migration-workflow.yaml:30]: Could this be shortened to some like this?
```
In-tree Volumes \[Driver: vsphere\] \[Testpattern: Inline-volume
In-tree Volumes \[Driver: vsphere\] \[Testpattern: Pre-provisioned PV
```

Comment (gnufied) [reply ci-operator/step-registry/openshift/e2e/vsphere/csi/migration/openshift-e2e-vsphere-csi-migration-workflow.yaml]: updated the regular expression after I realized, it has to be a single one. Not multiple lines.

**Comment by gnufied** [comment]
Given that job passed - https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/28104/rehearse-28104-periodic-ci-openshift-release-master-nightly-4.11-e2e-vsphere-techpreview/1518656012859478016

/hold cancel

**Pull request merged**